pkg/bgpv1/manager/manager.go

```diff
@@ -277,6 +277,7 @@ func (m *BGPRouterManager) registerBGPServer(ctx context.Context,
 	}
 
 	if err = m.reconcileBGPConfig(ctx, s, c, ciliumNode); err != nil {
+		s.Server.Stop()
```

Comment:
Doesn't [defer()](https://github.com/cilium/cilium/blob/main/pkg/bgpv1/manager/manager.go#L232-L239) set `err` and therefore stop the server?

Reply:
🤦 how did I miss that... Thanks for pointing it out. I'll close this PR.